src/pages/WalletsItemAddresses/components/WalletAddressCard/WalletAddressCard.js

```diff
@@ -66,13 +77,49 @@ class WalletAddressCard extends Component<Props, StateProps> {
     }
   }
 
+  handleChangeName = (event: SyntheticInputEvent<HTMLInputElement>) => {
+    this.setState({ newName: event.target.value })
+  }
+
+  handleActivateRename = (isRenameActive?: boolean = true) => {
+    this.setState({ isRenameActive: !!isRenameActive }, () => {
+      if (isRenameActive) {
+        this.nameInputRef.current.focus()
+      }
+    })
+  }
+
+  handleRenameBlur = () => {
+    this.handleActivateRename(false)
+
+    const {
+      address,
+      addressName,
+    } = this.props
+
+    const newName: string = this.state.newName.substring(0, 32).trim().replace(/\//g, '–')
```

Comment:
May be change order: `trim->replace->substring(...)`. Sometimes, this will allow to use strings with bigger length

Reply:
fixed in 374e4df